### DISCUSSION TITLE
Are TrinityCore Pull Requests stalled ?

### DISCUSSION THREAD

**Discussion by CraftedRO** [TrinityCore/TrinityCore#30297] [General]
Judging by the fact we have 90+ pull requests awaiting to be merged, I wonder what's going' on since no one merge them within reasonable time, are we waiting new year 😕 or ?

**Comment by Faq** [comment]
Not accurate, as few has [Feedback-FixOutdatedMissingWIP](https://github.com/TrinityCore/TrinityCore/pulls?q=is%3Apr+is%3Aopen+label%3AFeedback-FixOutdatedMissingWIP), but ye not small amount, what requires a lot time to review.

**Reply by CraftedRO** [reply]
Most likely 29026 and 28228 were good to go by the time when were posted.

**Comment by ghost** [comment]
They are stacking up. Does it require a lot of time to test or lack of interest?

**Comment by ImmortalusDev** [comment]
> Judging by the fact we have 90+ pull requests awaiting to be merged, I wonder what's going' on since no one merge them within reasonable time, are we waiting new year 😕 or ?

It was alwas like that, my friend.

**Comment by CraftedRO** [comment]
![667-square-H02_500](https://github.com/user-attachments/assets/cfde0822-28a9-40cb-9119-45947755846c)
